Author: Fabian Greffrath <fabian@debian-unofficial.org>
Description: Do not try to include Makefiles for ace or unace.dll
 and force remove in clean rule.

--- unace-nonfree-2.5.orig/base.mk
+++ unace-nonfree-2.5/base.mk
@@ -427,9 +427,7 @@ BASE_OS2_OBJS                     =
 ##############################################################################
 
 ifndef ISWMAKE
-include $(MAKE_DIR)baseace.mk
 include $(MAKE_DIR)baseuace.mk
 else
-!include $(MAKE_DIR)baseace.mk
 !include $(MAKE_DIR)baseuace.mk
 endif
--- unace-nonfree-2.5.orig/makefile
+++ unace-nonfree-2.5/makefile
@@ -29,8 +29,8 @@ endif
 all: unace
 
 clean:
-	rm unace
-	rm linunace25.tgz
+	rm -f unace
+	rm -f linunace25.tgz
 	
 include $(MAKE_DIR)base.mk
 include $(MAKE_DIR)apps.mk
--- unace-nonfree-2.5.orig/unace.mk
+++ unace-nonfree-2.5/unace.mk
@@ -27,8 +27,6 @@ APPS_UNACE_OBJS                     =
 
 ifndef ISWMAKE
 include $(MAKE_DIR)unaceexe.mk
-include $(MAKE_DIR)unacedll.mk
 else
 !include $(MAKE_DIR)unaceexe.mk
-!include $(MAKE_DIR)unacedll.mk
 endif
